README.md

```diff
@@ -201,6 +212,14 @@ The following documentation is available:
 <a name="known-issues"></a>
 ## Known Issues
 
+<a name="bugs"></a>
+### Bugs
+
+For a list of current bugs, see the [release notes](https://github.com/minishift/minishift/releases).
+
+<a name="unsupported-features"></a>
+### Unsupported features
```

Comment:
Like I mentioned, the text was already there so I just gave it a title. If you have a suggestion on how to frame this, based on your knowledge as to whether we plan to support these or not, please suggest it.